src/ontology/wbphenotype-edit.owl

```diff
@@ -2626,6 +2627,14 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://pu
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/WBPhenotype_0000002> "kinker"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/WBPhenotype_0000002> <http://purl.obolibrary.org/obo/WBPhenotype_0004017>)
 
+# Class: <http://purl.obolibrary.org/obo/WBPhenotype_0000003> (<http://purl.obolibrary.org/obo/WBPhenotype_0000003>)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/WBPhenotype_0000003> "obsolete"^^xsd:string)
```

Comment:
This is the human readable definition property.. Should this be there at all?

Reply:
@matentzn 
I was just following the trend that Gary had been using. Sue Bello had also said that she appends "obsolete" to the beginning of the definition for all obsolete MP terms, so I was playing around to see if this would be appropriate.